update_automation_example.py

```diff
@@ -0,0 +1,122 @@
+    )
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-u', '--url', help='Base URL of API', required=True)
+    parser.add_argument(
+        '-t', '--test', help='run unit test instead of updating automation', default=False, action='store_true')
+    parser.add_argument('-n', '--name', default="test", help='automation name')
+    parser.add_argument('-f', '--file', nargs='?', type=argparse.FileType('r'),
+                        help='automation script name(.py or .js)',
+                        required=False,
+                        default=sys.stdin)
+    options = parser.parse_args()
+    if options.name is None:
+        if options.file.name == '<stdin>':
+            if options.test:
+                options.name = "test"
+            else:
+                raise RuntimeError(
+                    'Error parsing options - name is required')
+        else:
+            options.name = os.path.splitext(
+                os.path.basename(options.file.name))[0]
+    return options
+
+
+class TestAutomation(unittest.TestCase):
+    def setUp(self):
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.id = automations["scripts"][0]["id"]
+
+    def tearDown(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        self.client.DeleteAutomation(automation)
+        self.id = None
+
+    def test_search(self):
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 1)
+        self.assertTrue("id" in automations["scripts"][0])
+
+    def test_delete(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 0)
+
+    def test_save_new(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 1)
+        self.assertTrue("id" in automations["scripts"][0])
+        self.id = automations["scripts"][0]["id"]
+
+    def test_load(self):
+        automation = self.client.LoadAutomation(self.id)
+        self.assertTrue("id" in automation)
+        self.assertTrue(automation["id"] == self.id)
```

Comment:
```python
self.assertEqual(automation["id"], self.id)
```

Reply:
use assertEqual instead of assertTrue(ea1a450).